GoogleScholarPlugin.php

```diff
@@ -115,7 +115,7 @@ public function submissionView($hookName, $args)
         $authors = $publication->getData('authors');
         foreach ($authors as $i => $author) {
             $templateMgr->addHeader('googleScholarAuthor' . $i, '<meta name="citation_author" content="' . htmlspecialchars($author->getFullName(false, false, $publicationLocale)) . '"/>');
-            if ($affiliation = $author->getLocalizedData('affiliation', $publicationLocale)) {
+            if ($affiliation = $author->getLocalizedAffiliationNamesAsString($publicationLocale, ', ')) {
```

Comment:
Hi @GaziYucel, lets take here several citation_author_institution elements, one for each affiliation.
I asked GoogleScholar how we should do it, but who knows if I will get an answer and when.
I found an SciELO example and they also use several element: view-source:https://www.scielo.cl/scielo.php?script=sci_arttext&pid=S1726-569X2023000200301&lng=en&nrm=iso&tlng=es.
I do not like us concatenating the affiliation names -- then it could be misstaken with a more detailed affiliation, containing e.g. university and department name -- so I would rather use it differently wherever possible...